test/integration/patients/worklist/schedule.js

```diff
@@ -665,29 +638,23 @@ context('schedule page', function() {
       .wait('@routeActions')
       .itsUrl()
       .its('search')
-      .should('contain', 'filter[workspace]=1')
       .should('contain', 'filter[@insurance]=Medicare');
 
     cy
       .get('.list-page__filters')
-      .find('[data-all-filters-region]')
-      .should('contain', '2');
-
-    cy
-      .get('.list-page__filters')
-      .find('[data-workspace-filter-region]')
-      .should('contain', 'Group One');
+      .find('.qa-all-filters')
+      .should('contain', '1');
 
     cy
       .get('.list-page__filters')
-      .find('[data-all-filters-region]')
+      .find('.qa-all-filters')
```

Comment:
If we don't want to use a `.qa-` selector here, I could do this instead:

```js
cy
  .get('[data-filters-region]')
  .find('button')
```

Reply:
yep better without it